roles/kubernetes/node/templates/node-kubeconfig.yaml.j2

```diff
@@ -4,6 +4,7 @@ clusters:
 - name: local
   cluster:
     certificate-authority: {{ kube_cert_dir }}/ca.pem
+    server: http://127.0.0.1:8080
```

Comment:
It does not work on none master node the API running on master nodes. 
So the url must go to http://master-node:8080 no ?

(possible to have multiple url server for multiple master node in the file ?)